packages/workflow-diagram/test/fixtures/single-workflow-nodeedges.json

```diff
@@ -309,7 +309,7 @@
       "labelBgStyle": {
         "fill": "#f3f4f6"
       },
-      "className": "dashed-edge"
+      "className": ""
```

Comment:
Surprised at this diff - I don't expect any changes here. I can't see where dashedEdge is actually assigned so this feels correct, but I'm concerned about these changes in general